circuit/src/nullifier.rs

```diff
@@ -131,20 +157,22 @@ impl CircuitFragment for Nullifier {
 pub mod test_helpers {
     use super::{Nullifier, NullifierInputs};
 
-    pub const PREIMAGE: &str =
+    pub const SECRET: &str =
         "776f726d686f6c650908804f8983b91253f3b2e4d49b71afc8e2c707608d9ae456990fb21591037f";
-
+    pub const FUNDING_NONCE: u32 = 0;
+    pub const FUNDING_ACCOUNT: &str = "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa";
     impl Default for Nullifier {
         fn default() -> Self {
-            let preimage = hex::decode(PREIMAGE).unwrap();
-            Self::new(&preimage)
+            let secret = hex::decode(SECRET).unwrap();
+            let account = hex::decode(FUNDING_ACCOUNT).unwrap();
+            Self::new(secret.as_slice(), FUNDING_NONCE, <[u8; 32]>::try_from(account.as_slice()).unwrap())
```

Comment:
```suggestion
            Self::new(secret.as_slice(), FUNDING_NONCE, default_account_bytes())
```
Probably cleaner to have something like this instead. We can then reuse the same function in the prover tests, etc.